.github/workflows/bundle_with_dakota_caller.yml

```diff
@@ -0,0 +1,113 @@
+
+    steps:
+      - uses: actions/cache@v3
+        name: Check cache for already built wheels
+        id: cache-package
+        with:
+          key: carolina_wheels_boost${{ env.BOOST_VERSION }}_dakota${{ env.DAKOTA_VERSION }}_seba${{ env.SEBA_TAG }}
+          path: /tmp/dist
+      - name: Write cache status to outputs
+        if: steps.cache-package.outputs.cache-hit == 'true'
+        run:
+          echo NEEDS_REBUILD=false >> $GITHUB_OUTPUT
+
+  build_macos_wheels:
+    needs: build-settings
+    if: ${{ needs.build-settings.outputs.NEEDS_REBUILD == 'true' }}
+    uses: ./.github/workflows/bundle_with_dakota_macos.yml
+    with:
+      ERT_SHOW_BACKTRACE: ${{ needs.build-settings.outputs.ERT_SHOW_BACKTRACE}}
+      NO_PROJECT_RES: ${{ needs.build-settings.outputs.NO_PROJECT_RES}}
+      BOOST_VERSION: ${{ needs.build-settings.outputs.BOOST_VERSION}}
+      BOOST_VERSION_UNDERSCORES: ${{ needs.build-settings.outputs.BOOST_VERSION_UNDERSCORES}}
+      DAKOTA_VERSION: ${{ needs.build-settings.outputs.DAKOTA_VERSION}}
+      SEBA_TAG: ${{ needs.build-settings.outputs.SEBA_TAG}}
+      INSTALL_DIR: ${{ needs.build-settings.outputs.INSTALL_DIR}}
+    secrets: inherit
+
+  build_linux_wheels:
+    if: ${{ needs.build-settings.outputs.NEEDS_REBUILD == 'true' }}
+    needs: build-settings
+    uses: ./.github/workflows/bundle_with_dakota_linux.yml
+    with:
+      ERT_SHOW_BACKTRACE: ${{ needs.build-settings.outputs.ERT_SHOW_BACKTRACE}}
+      NO_PROJECT_RES: ${{ needs.build-settings.outputs.NO_PROJECT_RES}}
+      BOOST_VERSION: ${{ needs.build-settings.outputs.BOOST_VERSION}}
+      BOOST_VERSION_UNDERSCORES: ${{ needs.build-settings.outputs.BOOST_VERSION_UNDERSCORES}}
+      DAKOTA_VERSION: ${{ needs.build-settings.outputs.DAKOTA_VERSION}}
+      SEBA_TAG: ${{ needs.build-settings.outputs.SEBA_TAG}}
+      INSTALL_DIR: ${{ needs.build-settings.outputs.INSTALL_DIR}}
+    secrets: inherit
+
+  publish_wheels:
+    needs: [ build-settings, build_macos_wheels, build_linux_wheels ]
+    runs-on: ubuntu-latest
+    steps:
+      - name: "READ THE ARTIFACT"
+        uses: actions/download-artifact@v3
+        with:
+          path: artifacts
+      - name: "ls the artifacts dir"
+        run: | 
+          
+      - name: Create dist/ folder with all the wheels
+        run: |
+          ls artifacts
+          ls artifacts/artifact
+          pwd
+          mkdir /tmp/dist
+          find artifacts -type f -name "carolina*.whl" -exec mv {} /tmp/dist/ \;
+          ls /tmp/dist
+      - name: Cache wheels folder
+        if: steps.cache-package.outputs.cache-hit != 'true'
+        uses: actions/cache/save@v3
+        with:
+          path: /tmp/dist
+          key: carolina_wheels_boost${{ needs.build-settings.outputs.BOOST_VERSION }}_dakota${{ needs.build-settings.outputs.DAKOTA_VERSION }}_seba${{ needs.build-settings.outputs.SEBA_TAG }}
+      
+      - name: Publish to pypi
+        uses: pypa/gh-action-pypi-publish@v1.8.10
+        if: >
```

Comment:
Unsure what this does, can you elaborate?

Reply:
Since it is a boolean expression, it makes sense to "inline" it over multiple lines; the `>` indicator replaces newlines by spaces, as opposed to keeping them with the `|` indicator

Reply:
Thanks! I never knew that!